README.md

```diff
@@ -164,6 +178,42 @@ Replace `${ACCESS_KEY}` with yours obtained from Picovoice Console and `${MODEL_
 [default model file](/lib/common/cheetah_params.pv) (or your custom one). Finally, when done be sure to release
 resources acquired using `pv_cheetah_delete(handle)`.
 
+### iOS
+
+The Cheetah iOS binding is available via [Cocoapods](https://cocoapods.org/pods/Cheetah-iOS). To import it into your iOS project, add the following line to your Podfile and run `pod install`: 
+
+```ruby
+pod 'Cheetah-iOS'
+```
+
+Create an instance of the engine and transcribe audio in real-time:
+
+```swift
+import Cheetah
+
+let modelPath = Bundle(for: type(of: self)).path(
+        forResource: "${MODEL_FILE}", // Name of the model file name for Cheetah
+        ofType: "pv")!
+
+let cheetah = Cheetah(accessKey: "${ACCESS_KEY}", modelPath: modelPath)
+
+func getNextAudioFrame() -> [Int16] {
+  // .. get audioFrame
+  return audioFrame;
+}
+
+while true {
+  do {
+    let partialTranscript, isEndpoint = try cheetah.process(getNetAudioFrame())
+    if isEndpoint {
+      let finalTranscript = try cheetah.flush()
+    }
+  } catch { }
+}
+```
+
+Replace `${ACCESS_KEY}` with yours obtained from Picovoice Console and `${MODEL_FILE}` with the name of the Cheetah model file name.
```

Comment:
default or custom trained from Console (something like that and maybe a link to default model)